crates/libs/windows/src/Windows/Win32/Graphics/Direct3D12/mod.rs

```diff
@@ -19830,9 +19842,9 @@ impl ID3D12Device {
         ::core::mem::transmute((::windows::core::Interface::vtable(self).GetDescriptorHandleIncrementSize)(::core::mem::transmute_copy(self), ::core::mem::transmute(descriptorheaptype)))
     }
     #[doc = "*Required features: 'Win32_Graphics_Direct3D12'*"]
-    pub unsafe fn CreateRootSignature<T: ::windows::core::Interface>(&self, nodemask: u32, pblobwithrootsignature: *const ::core::ffi::c_void, bloblengthinbytes: usize) -> ::windows::core::Result<T> {
+    pub unsafe fn CreateRootSignature<T: ::windows::core::Interface>(&self, nodemask: u32, pblobwithrootsignature: &[u8]) -> ::windows::core::Result<T> {
```

Comment:
This function is usually used in conjunction with ID3DBlob with GetBufferPointer() and GetBufferSize().  Will this interop nicely with the array parameter?

Reply:
I updated your Direct3D sample (in this repo) to use it as follows:

```rust
device.CreateRootSignature(0, std::slice::from_raw_parts(signature.GetBufferPointer() as _, signature.GetBufferSize())) 
```

Reply:
hmm....not beautiful, but could probably be fixed with a simple helper to build a slice from a ID3DBlob directly.